ci-operator/step-registry/openshift/microshift/e2e/pre-rpm-install/openshift-microshift-e2e-pre-rpm-install-commands.sh

```diff
@@ -32,10 +32,21 @@ gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
 gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
 gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
 
+
+# The dnf-automatic systemd unit holds a lock on /var/lib/.rpm.lock on first-boot.
+# Wait until the service completes the system upgrade before performing further rpm/dnf operations.
+# Fixes error: 
+LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+  --zone "${GOOGLE_COMPUTE_ZONE}" \
+  rhel8user@"${INSTANCE_PREFIX}" \
+  --command 'while $(sleep 5); do sudo systemctl is-active --quiet dnf-automatic || break; echo "waiting for systemd unit dnf-automatic" to complete; done'
```

Comment:
should we just kill it? }:)

Reply:
The process is running security and other OS package updates.  Due to cumulative run time factors like pod startup and network latency with the instance, it couldn't be consistently killed in time, and could produce some very-hard-to-detect OS variability between each test run.

Reply:
Why would there be os variability if the image is always the same?

shall we disable this service in the image then? (and make sure the image is updated periodically?

Reply:
That's the question - the image is identical for all jobs. So why is it that the dnf-automatic service takes anywhere between 1 minute and ~30 minutes to complete?